GOVERNANCE.md

```diff
@@ -0,0 +1,146 @@
+
+A **binding vote** is one cast by specific individuals with given roles based on the voting procedure, who's vote is used to calculate the outcome.
+A **non-binding vote** is one cast by individuals who do not have binding vote roles, and these votes do not count toward the outcome, but are used to guide those with binding votes to a resolution that best fits all involved.
+
+The following describes for each procedure, those who may vote, how the votes are recorded, and duration of the vote.
+
+### Pull request approvals
+
+Pull requests against most repositories may be merged after receiving at least two positive committer votes.
+Voting for pull requests can be done using a comment or approving the pull request on GitHub.
+If the pull request author is a member of the Debezium steering committee, this counts as one of the two required votes.
+
+Pull requests that are small, low-risk, uncontroversial, or prefixed with `[ci]` or `[docs]` may freely be merged by any committer without any votes.
+Examples of uncontroversial changes may include:
+
+* Refactoring redundant code within a small number (2-5) of files that yields no behavior change.
+* Adding or changing logged output.
+* Adding or improving test suite code with no loss of test coverage.
+
+Whenever in doubt about the pull request's scope and risk, use the two positive vote requirement.
+
+### Lazy consensus votes
+
+Lazy consensus votes use the **+1**, **0**, or **-1** votes where:
+
+* `+1` indicates **yes** or the **affirmative**
+* `-1` indicates **no** or the **negative**
+* `0` indicates you have no specific opinion
+
+The voting process concludes when there are at least three `+1` binding votes, and no `-1` binding votes.
+The vote should be open for at least **TBD** days to allow everyone to participate.
+
+### Explicit majority votes
+
+An explicit majority vote is simply a `+1` or `-1` based vote.
+The vote succeeds when at least **three binding** votes are cast, and two-thirds of cast binding values are `+1`.
+The vote should be open for at least **TBD** days to allow everyone to participate.
+
+## Voting for specific subject areas
+
+The following describes the voting procedures for specific subject areas.
+
+### Governance changes
+
+Except for the steering committee members marking themselves as Emeritus, all changes in the [governance](https://github.com/debezium/governance) repository require an explicit majority vote of steering committee members.
+These votes are held by the steering committee, and the outcome will be publicly published.
+
+### Add/remove steering committee members
+
+Votes to add or remove steering committee members require an explicit majority vote of steering committee members.
+These votes are held by the steering committee, and the outcome will be publicly published.
+
+If a steering committee member is absent for more than 6-months without any prior notification of absence, the committee member will be moved to Emeritus status automatically.
```

Comment:
This feels a bit harsh tbh. What exactly qualifies as absence? I think there should be a step first where the committee checks in, and only if they don't hear back in some time, the Emeritus status is achieved.

Reply:
What sort of turn-around would you say is reasonable for that check-in?  1 month or something less?

Reply:
From my PoV 1 month check is perfect.
Absence should be deined as non-communication and no activity in the porject or project org.
Pracially I can imagine it will take more than 6 months as I don't think we'll have a tool to monitor this so it will take some time before somebody notices and raise the matter first.

Reply:
Agreed, a 1 month check-in / pulse check after the 6 months before moving to Emeritus.